### ISSUE TITLE
Citation

### ISSUE THREAD

**Issue by alandefreitas** [olivierbbb/nodesoup#4]
Hi,

I'd like to mention your repository on a paper.

Is there a name I can mention?
Is the official library name "nodesoup" or "NodeSoup"?

Thanks,

**Comment by olvb** [comment]
Hi, thanks for the mention! The name is Olivier Birot and the library name is nodesoup.

**Comment by alandefreitas** [comment]
Thanks. I've updated the paper. It has not been reviewed yet but I'll let you know when it comes out.
I've also fixed the reference I made to nodesoup in our repository: https://github.com/alandefreitas/matplotplusplus/commit/49b660511b8859d65356ce4a98b9fa963b941faf
Nodesoup is being used for this plot category: https://github.com/alandefreitas/matplotplusplus#graphs
If there's anything else you'd like me to mention or anything you don't like with the references, just let me know.
Thanks for the library!
